packages/components/src/Card/Card.tsx

```diff
@@ -70,7 +70,7 @@ export function Card({ accent, children, url, onClick }: CardPropOptions) {
     href?: string;
     role?: "button";
     tabIndex?: 0;
-    onClick?(): void;
+    onClick?(e: React.MouseEvent<HTMLElement>): void;
```

Comment:
thoughts on naming this `event` instead of `e`? I know `e` is also used to convey event but it could also mean a lot of other things. Remove the FUD even before it happens

> The best icon to represent edit is to put the word edit on it